tests/integration/test_backup_restore_on_cluster/configs/disallow_concurrency.xml

```diff
@@ -0,0 +1,15 @@
+<clickhouse>
+    <storage_configuration>
+        <disks>
+            <backups>
+                <type>local</type>
+                <path>/backups/</path>
+            </backups>
+        </disks>
+    </storage_configuration>
+    <backups>
+        <allowed_disk>backups</allowed_disk>
+    </backups>
+    <allow_concurrent_backups>false</allow_concurrent_backups>
+    <allow_concurrent_restores>false</allow_concurrent_restores>
```

Comment:
For me the term `concurrent backups` looks like if you're making two backups of the same resource (e.g. the same table) at the same time. For the thing you constrained in this PR the name `parallel backups` probably would be more clear.

Reply:
For this setting, I feel 'concurrent' is better than 'parallel'. The constraint is on running more than 1 backup/restore at the same time irrespective of the name. As we already refer to this as 'concurrent', I think it would be confusing to call the constraint on it 'parallel'. Please let me know your thoughts on this.

Reply:
AFAIK the term was never referred in the code or settings or configuration as concurrent or parallel so it's quite a random choice.

Reply:
Its not in settings or configuration, but we have error messages like 
https://github.com/ClickHouse/ClickHouse/blob/67e2bf31f5a0f5c9313e0fe78e56c51716a5b082/src/Backups/BackupImpl.cpp#L501
and tests named `test_concurrency`, so to keep in line with existing structure I used 'concurrent'.